radionuclide_scraper.py

```diff
@@ -261,102 +254,91 @@ def extract_half_life(table, index):
     if half_life_text[0] == '<':
         half_life_text = half_life_text[1:]
         half_life_text = half_life_text.strip()
-    
-    
+
     half_life = float(half_life_text)
 
     if time_symbol == 'm':
         half_life_seconds = 60*half_life
     elif time_symbol == 'h':
         half_life_seconds = 3600*half_life
-    elif time_symbol == 'd': 
+    elif time_symbol == 'd':
         half_life_seconds = 3600*24*half_life
     elif time_symbol == 'y':
         half_life_seconds = (3.154*10**7)*half_life
-    
+
     decay_constant_seconds = (math.log(2))/(half_life_seconds)
 
     return(half_life_seconds, decay_constant_seconds)
-    
-    
-    
+
+
 def extract_atomic_mass_numbers(url):
-    
     """
     Acquiring atomic number and mass number.
     """
-    
     numbers = ''
-    
+
     for i in range(len(url)):
         if url[i] == '=':
             numbers = numbers + url[i+1:]
             numbers = numbers
             mass_number = int(numbers[-3:])
             atomic_number = numbers[:-3]
             break
-    
-    #For mass number.    
+    """
+    For mass number.
+    """
     if mass_number > int(300):
         mass_number = mass_number - 300
-    
-    #For atomic number.
+    """
+    For atomic number.
+    """
     if len(atomic_number) == 2:
         atomic_number = int(atomic_number[0])
     elif len(atomic_number) == 3:
         atomic_number = int(atomic_number[0:2])
     elif len(atomic_number) == 4:
         atomic_number = int(atomic_number[0:3])
-    
+
     return(atomic_number, mass_number)
-    
-    
-
-class Isotope(object):
-    def __init__(self, symbol, atomic_number, mass_number, half_life, decay_constant, list_sig_g_e, list_sig_g_b_r):
-        self.symbol = symbol
-        self.atomic_number = atomic_number
-        self.mass_number = mass_number
-        self.half_life = half_life
-        self.decay_constant = decay_constant
-        self.list_sig_g_e = list_sig_g_e
-        self.list_sig_g_b_r = list_sig_g_b_r
-        return
-        
-    
-    
+
+
 def compile_objects(info):
-    
     """
     Returns a list of objects.
     """
-    
-    database = []        
+    database = []
     for i in range(len(info)):
-        vars()[info[i][0] + '_' + str(info[i][2])] = Isotope(info[i][0],info[i][1],info[i][2],info[i][3],info[i][4],info[i][5],info[i][6])
+        vars()[info[i][0] + '_' + str(info[i][2])] = Isotope(info[i][0],
+                                                             info[i][1],
+                                                             info[i][2],
+                                                             info[i][3],
+                                                             info[i][4],
+                                                             info[i][5],
+                                                             info[i][6])
```

Comment:
Good indentation, but there is a shortcut for this kind of argument unpacking. You can do `Isotope(*info[i])` and it should unpack all the elements of `info[i]` in order, like you are doing manually right now.